Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:2.7
+
+COPY . /securitybot
+
+env PYTHONPATH $PYTHONPATH:/securitybot
+
+RUN apt-get update && \
+    apt-get install -y mysql-client && \
```

Comment:
Perhaps rather than installing mysql in this container we could link to a separate mysql container, and include a `docker-compose file.`